app/controllers/assign_mentor.rb

```diff
@@ -0,0 +1,10 @@
+class AssignMentor
```

Comment:
Is there a unit test for this class?

Reply:
Yes, it was in the controller tests, just moved it to it's own file